src/main/java/com/example/HR_AppJava/HR_App/Job.java

```diff
@@ -9,13 +9,18 @@ public class Job {
     private String title;
     private int experiance;
 
-    public void setTitle(){
-        System.out.println("Enter title: ");
+    public Job(){
```

Comment:
You shouldn't call your modifier methods inside of your constructor. For the default constructor, you just need 
public Job(){}
You could also add a constructor that takes parameters as
public Job(String title, int experience){
this.title = title;
this.experience = experience;
}